src/EventHandlers/HandlesEvents.php

```diff
@@ -13,18 +13,41 @@ public function handlesEvents(): array
 
     public function handlesEvent(object $event): bool
     {
-        return array_key_exists(get_class($event), $this->handlesEvents());
+        $handlesEvents = $this->handlesEvents();
+
+        return array_key_exists(get_class($event), $handlesEvents)
+            || isset($handlesEvents);
```

Comment:
This doesn't seem right. The function will make the function always return `true`

Reply:
OK.  I'll take a look....and perhaps introduce a test to force a `false`.